packages/twenty-server/src/database/typeorm-seeds/core/demo/feature-flags.ts

```diff
@@ -1,26 +1,17 @@
 import { DataSource } from 'typeorm';
 
-import { FeatureFlagKeys } from 'src/core/feature-flag/feature-flag.entity';
-
 const tableName = 'featureFlag';
 
 export const seedFeatureFlags = async (
   workspaceDataSource: DataSource,
   schemaName: string,
-  workspaceId: string,
```

Comment:
Why is it removed?

Reply:
the variable is unused so the linter was complaining. We will re-add it when needed

Reply:
actually I have commented the code so I reverted this change